### PR TITLE
Add client properties information

### LINKED ISSUE #2: Add documentation for key values on the local saves
Including:

- gamesave properties
- level properties
- object properties
- guildelines

### PR DESCRIPTION
**Pull request by AlFasGD**
Closes #2

### CODE REVIEW

**Comment by WOSHIZHAZHA120** [comment]
Have a conflicts, pls resolved.

File: docs/_sidebar.md [hunk 1]
```diff
@@ -10,7 +10,12 @@
   - [Level](/resources/server/level.md)
   - [User](/resources/server/user.md)
 - **Client**
+  - [Gamesave](/resources/client/gamesave.md)
   - [Level](/resources/client/level.md)
+  - [Inner Level String](/resources/client/inner-level-string.md)
+  - [Level Object](/resources/client/level-object.md)
+  - [Color String](/resources/client/color-string.md)
+  - [Guideline String](/resources/client/guideline-string.md)
```
Comment (Altenhh) [inline docs/_sidebar.md:18]: These can be grouped into their own section (preferably under Level)

File: docs/resources/client/guideline-string.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
+# Client Guideline String Resource
+
+## Guideline String
+The guideline string is the linearized string format of the guidelines of a level. It is contained within the [inner level string](). It is formatted as follows:
```
Comment (Altenhh) [inline docs/resources/client/guideline-string.md:4]: Please edit all the links that you have and provide them with actual redirects (if any).

File: docs/resources/client/guideline-string.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+`{guideline}~{guideline}~{guideline}~...`, where
+
+`guideline` is formatted as:
```
Comment (Altenhh) [inline docs/resources/client/guideline-string.md:8]: This can be in the same line.

File: docs/resources/client/inner-level-string.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+- `object_string` is the [object string]().
+
+Theoretically, the inner level string in its entirety is the object string, however the level start object is treated specially, unlike every other object, and doesn't even have an ID.
+
+## Level Start Object
+The level start object is still an object and formated exactly like a normal [level object](level-object.md), but has the following properties instead:
+
+| Key  | Name                   | Type                                        | Description                                                                                                    |
+|:-----|:-----------------------|:--------------------------------------------|:---------------------------------------------------------------------------------------------------------------|
+| kA2  | Gamemode               | **[Gamemode](enumerations.md)**             | the gamemode the player starts with                                                                            |
```
Comment (Altenhh) [inline docs/resources/client/inner-level-string.md:18]: Where is this enumerations document?

File: docs/resources/client/level-object.md [hunk 5]
```diff
@@ -0,0 +1,189 @@
...
+
+## Level Object
+A level object is an object within a Geometry Dash level, containing data about its behavior.
+
+### Object String
+The object string is contained within the [inner level string](). It is formatted as follows:
+
+`{object};{object};{object};...`, where
+
+`object` is formatted as:
```
Comment (Altenhh) [inline docs/resources/client/level-object.md:11]: Can be formatted into a single line.

File: docs/resources/client/inner-level-string.md (hunk 4 above)
Comment (AlFasGD) [reply docs/resources/client/inner-level-string.md:18]: Will be made in another PR

**Comment by AlFasGD** [comment]
> Have a conflicts, pls resolved.

Holy shit you're a genius

**Comment by SMJSGaming** [comment]
ah ye, likely me fixing the caps in most files

**Comment by SMJSGaming** [comment]
tried fixing the conflict, idk what tf it wants but maybe try requesting it again because github atm sees the file itself as a conflict which makes no sense

**Comment by SMJSGaming** [comment]
looks good

**Pull request merged**